src/main/util.py

```diff
@@ -81,14 +108,17 @@ def find_matching_release(release_tags: list[str], version: str) \
             and "API rate limit exceeded" in response.json()["message"]:
         reset_time = response.headers.get("X-RateLimit-Reset")
 
+        wait_time: float = int(reset_time) - time()
+
         # Convert the reset time to a human-readable format
         reset_time = datetime.datetime.fromtimestamp(int(reset_time))
         reset_time = reset_time.strftime("%Y-%m-%d %H:%M:%S")
-
-        raise ConnectionRefusedError(
-            (f"GitHub API rate limit exceeded. Try again later."
-             f"Rate limit resets at {reset_time}.")
-            )
+        print(f"GitHub API rate limit exceeded. Rate limit resets at {reset_time}.")
```

Comment:
Perhaps not a print statement this deep down, could ruin something when you want to put it in a pipeline

Reply:
Is it better to just leave it going? Thought it might be confusing, but can remove the print statement ^^

Reply:
Removed print statement now ^^